packages/declaration/lua/language-server/global.def.lua

```diff
@@ -64,3 +67,7 @@ function destroyObject(object) end
 ---@param message table
 ---@return boolean
 function sendExternalMessage(message) end
+
+---@param function_owner tts__GameObject
```

Comment:
This should be `tts__Object` instead. Game Object is the stuff that is returned by the component API (https://api.tabletopsimulator.com/components/gameobject/).